src/opera_disp_tms/__main__.py

```diff
@@ -39,7 +40,7 @@ def generate_tile_map_service(
         measurement_geotiff = create_measurement_geotiff(measurement_type, frame, begin_date, end_date)
         measurement_geotiffs.append(measurement_geotiff.name)
 
-    # FIXME sort geotiffs to give the layering we want
+    measurement_geotiffs = sort_geotiffs(measurement_geotiffs)
```

Comment:
Implementing a sort key function instead of a sorting function would likely make testing easier:
```
measurement_geotiffs.sort(sort_key=lambda x: get_geotiff_sort_key(x))
```